packages/kit/test/apps/basics/src/routes/before-navigate/preserve-search-params/+layout.svelte

```diff
@@ -0,0 +1,22 @@
+<script>
+	import { beforeNavigate } from '$app/navigation';
+
+    // do not change, used in test (/basics/test/cross-platform/client.test.js):
+    const preserved_param = 'preserved';
+    const preserved_param_value = 'true';
```

Comment:
```suggestion
```

This doesn't seem to be used, unless it was intended for line 11?

Reply:
@ghostdevv yes, you're totally right!